api/config/.env.local

```diff
@@ -0,0 +1 @@
+DBHost=http://localhost:5432
```

Comment:
Should it be a new line here since it is the end of a file?

Reply:
Yes! Good catch! 👌 Will update later this evening.

Reply:
I realized that this file should even be removed as it's not used anymore.

Reply:
Great!